FirebaseAppCheck/Sources/Public/FirebaseAppCheck/FIRAppCheck.h

```diff
@@ -52,36 +54,6 @@ NS_SWIFT_NAME(AppCheck)
 /// @throw Throws an exception if required `FirebaseApp` options are missing.
 + (nullable instancetype)appCheckWithApp:(FIRApp *)firebaseApp NS_SWIFT_NAME(appCheck(app:));
 
-/// Requests Firebase app check token. This method should *only* be used if you need to authorize
```

Comment:
Sorry I missed it in the API review, but why is it ok to remove public APIs in a minor version update?

Reply:
These APIs were copied into `FIRAppCheckProtocol` and `FIRAppCheck` now conforms to it. The API diffing tool is a bit misleading in its report that the APIs were deleted.

As a consistency check, https://github.com/firebase/firebase-ios-sdk/blob/f3c7db784532b956a9d0d7134cae24730b758e0e/FirebaseAppCheck/Tests/Unit/Swift/AppCheckAPITests.swift is unchanged. Also verified locally that autocomplete behaves as before in Xcode.

Reply:
Great thanks!

Reply:
cc: @sunmou99 for the API diff report not taking into account the new protocol conformance making up for the removed API